besu/src/main/java/org/hyperledger/besu/cli/BesuCommand.java

```diff
@@ -446,6 +448,39 @@ void setBannedNodeIds(final List<String> values) {
           "Require authentication for the JSON-RPC HTTP service (default: ${DEFAULT-VALUE})")
   private final Boolean isRpcHttpAuthenticationEnabled = false;
 
+  @Option(
+      names = {"--rpc-http-tls-enabled"},
+      description = "Enable TLS for the JSON-RPC HTTP service (default: ${DEFAULT-VALUE})")
+  private final Boolean isRpcHttpTlsEnabled = false;
+
+  @Option(
+      names = {"--rpc-http-tls-keystore-file"},
+      paramLabel = MANDATORY_FILE_FORMAT_HELP,
+      description =
+          "Path to PKCS#12 format key store for the JSON-RPC HTTP service. Required if TLS is enabled.")
+  private final File rpcHttpTlsKeystoreFile = null;
+
+  @SuppressWarnings("FieldMayBeFinal") // Because PicoCLI requires Strings to not be final.
+  @Option(
+      names = {"--rpc-http-tls-keystore-password"},
```

Comment:
do we want to make this a "path" rather than a plain-text password? Or am I mistaking this field's purpose?
This probably applies below too

Reply:
@rain-on  Password is not read from a file, it is specified on cli as in `--rpc-http-tls-keystore-password=changeit`

Reply:
Yup - so in Ethsigner we deliberately used password-path such that passwords didn't appear in a launch script, is it safe for this password to be in plain text on cmdline? Or should it be contained in a file, such that it isn't visible?